src/core/Hypridle.cpp

```diff
@@ -293,8 +293,13 @@ void CHypridle::onResumed(SIdleListener* pListener) {
 
 void CHypridle::onInhibit(bool lock) {
     m_iInhibitLocks += lock ? 1 : -1;
-    if (m_iInhibitLocks < 0)
+    if (m_iInhibitLocks < 0) {
         Debug::log(WARN, "BUG THIS: inhibit locks < 0");
+        // what would be safer appending one or setting to 0?
+        // what if would be equal -2?
+        m_iInhibitLocks++;
+        Debug::log(WARN, "APPEND THIS: inhibit locks ++");
```

Comment:
tf is "APPEND THIS"

Anyways just m_iInhibitLocks = 0 lol

Reply:
Better?

https://github.com/hyprwm/hypridle/pull/11/commits/5332b443e4bb4619301eaea36af59d0f88b67fc5

I stand by incrementing anyway. Safer. If you want definitely zeroing though...

Reply:
zero it, no point in going negative

Reply:
done